src/Events/Integrations/Plugins/Event_Tickets/Emails/Email/RSVP.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Class RSVP.
+ *
+ * @since   TBD
```

Comment:
```suggestion
 * @since TBD
```

Let's do a search replace on all the since tags with more than 1 space.

Reply:
I'd love that